packages/smooth_app/lib/themes/smooth_theme.dart

```diff
@@ -62,6 +62,14 @@ class SmoothThemes {
       ),
       //
       //
+      bottomNavigationBarTheme: BottomNavigationBarThemeData(
```

Comment:
You should create a BottomNavigationBarTheme in your dark/light themes instead, which would automatically be reused by all BottomNavigationBar.